src/BinSkim.Rules/PERules/BA6001.EnableSourceLink.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+
+using Microsoft.CodeAnalysis.BinaryParsers;
+using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
+using Microsoft.CodeAnalysis.IL.Sdk;
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    [Export(typeof(Skimmer<BinaryAnalyzerContext>)), Export(typeof(ReportingDescriptor)), Export(typeof(IOptionsProvider))]
+    public class EnableSourceLink : WindowsBinaryAndPdbSkimmerBase, IOptionsProvider
+    {
+        /// <summary>
+        /// BA6001
+        /// </summary>
+        public override string Id => RuleIds.EnableSourceLink;
+
+        /// <summary>
+        /// Enable SourceLink.
+        /// </summary>
+        public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = RuleResources.BA6001_EnableSourceLink_Description };
+
+        protected override IEnumerable<string> MessageResourceNames => new string[]
+        {
+            nameof(RuleResources.BA6001_Pass),
+            nameof(RuleResources.BA6001_Error)
+        };
+
+        private const string AnalyzerName = RuleIds.EnableSourceLink + "." + nameof(EnableSourceLink);
```

Comment:
>RuleIds.EnableSourceLink + "." + nameof(EnableSourceLink); [](http://example.com/codeflow?start=44&length=58)

If we established this code convention today, I think we'd use an interpolated string. Feel free to do that. We can go back later and clean everyone else up.

$"{RuleIds.EnableSourceLink}.{nameof(EnableSourceLink)};

Reply:
Const string interpolation requires C# 10
I copied this from some other rule, but it turns out it isn't used and can be deleted.